WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -292,7 +292,7 @@ public void onLog(T tag, LogLevel logLevel, String message) {
         // remove expired lists
         mDispatcher.dispatch(ListActionBuilder.newRemoveExpiredListsAction(new RemoveExpiredListsPayload()));
 
-        // setup the Credentials Client so we can clean it up on wpcom logout
+        // setup the Credentials Client so it can be cleaned on wpcom logout
```

Comment:
I believe this actually changed what the message was saying. The original was more assertive that _we do clean it up_ while this new comment is saying _"it can or might be cleaned up"_. 

Can we revert it? 🙂